examples/02_decoding/plot_haxby_glm_decoding.py

```diff
@@ -104,84 +103,62 @@
-
-# no need to standardize or smooth the data
-masker = NiftiMasker(mask_img=haxby_dataset.mask, memory='nilearn_cache',
-                     memory_level=1)
-X = masker.fit_transform(z_maps)
-
-#############################################################################
-# Build the decoder
-# ------------------
-# Define the prediction function to be used.
-# Here we use a Support Vector Classification, with a linear kernel
-from sklearn.svm import SVC
-
-svc = SVC(kernel='linear')
-
-# Define the dimension reduction to be used.
-# Here we use a classical univariate feature selection based on F-test,
-# namely Anova. When doing full-brain analysis, it is better to use
-# SelectPercentile, keeping 5% of voxels
-# (because it is independent of the resolution of the data).
-from sklearn.feature_selection import SelectPercentile, f_classif
-
-feature_selection = SelectPercentile(f_classif, percentile=5)
-
-# We have our classifier (SVC), our feature selection (SelectPercentile),and now,
-# we can plug them together in a *pipeline** that performs the two operations
-# successively:
-from sklearn.pipeline import Pipeline
-
-anova_svc = Pipeline([('anova', feature_selection), ('svc', svc)])
-
-#############################################################################
-# Obtain prediction scores via cross validation
-# -----------------------------------------------
-from sklearn.model_selection import cross_val_score, LeaveOneGroupOut
-
-# Define the cross-validation scheme used for validation.
-# Here we use a LeaveOneGroupOut cross-validation on the session group
-# which corresponds to a leave-one-session-out
-cv = LeaveOneGroupOut()
-
-# Compute the prediction accuracy for the different folds (i.e. session)
-cv_scores = cross_val_score(anova_svc, X, condition_idx, cv=cv, groups=session_idx)
-
-# Return the corresponding mean prediction accuracy
-classification_accuracy = cv_scores.mean()
-chance_level = 1. / len(np.unique(condition_idx))
-
-# Print the results
+##############################################################################
+# Build the decoding pipeline
+# ----------------------------
+# To define the decoding pipeline we use Decoder object, we choose :
+#
+# * a prediction model, here a Support Vector Classifier, with a linear kernel
+#
+# * the mask to use, here a ventral temporal ROI in the visual cortex
+#
+# * although it usually helps to decode better, z-maps time series don't need
+#    to be rescaled to a 0 mean, variance of 1 so we use standardize=False.
+#
+# * we use univariate feature selection to reduce the dimension of the problem
+#    keeping only 5% of voxels which are most informative.
+#
+# * a cross-validation scheme, here we use LeaveOneGroupOut cross-validation
+#    on the sessions which corresponds to a leave-one-session-out
+#
+# We fit directly this pipeline on the Niimgs outputs of the GLM, with
+# corresponding conditions labels and session labels (for the cross validation).
```

Comment:
> The remaining error reads :
> 
> ```
> sphinx.errors.SphinxWarning:/home/circleci/project/doc/auto_examples/02_decoding/plot_haxby_glm_decoding.rst:12481:Unexpected section title or transition.
> ----------------------------
> Warning, treated as error:
> /home/circleci/project/doc/auto_examples/02_decoding/plot_haxby_glm_decoding.rst:12481:Unexpected section title or transition.
> ```
> 
> With no more precisions and despite rereading the file 10 times and searching a bit on the internet, I don't know whats wrong. Anyone who knows numpydoc want to take a look, or at least explain me where this can come from ?



My guess is the error is related to formatting of this section with those asterisks `*`.

Reply:
Example ```02_decoding_plot_simulated_data.py``` has such asterisks definitions. Maybe you could adapt in the same way as there.